constants/constants.ts

```diff
@@ -160,11 +160,11 @@ export const SPELL_OWNER_ADDRESS = {
 };
 
 export const SAMB_ADDRESS = {
-  [CHAINID.ETH_MAINNET]: "0x278a8c33Ef2A7D7C682f458C85E9447ce3645188",
+  [CHAINID.ETH_MAINNET]: "0x683aae5cD37AC94943D05C19E9109D5876113562",
 };
 
 export const SAMB_OWNER_ADDRESS = {
-  [CHAINID.ETH_MAINNET]: "0x6d5b28659b6e76c5843c963f44158c3be6ede3a1",
+  [CHAINID.ETH_MAINNET]: "0x39f36935e07f283cf1fd58984c3ed2e63e8b5ccd",
```

Comment:
Since this address is mainly used for testing, I think we can leave it as `0x6d5b28659b6e76c5843c963f44158c3be6ede3a1`. This address has a balance of 10,000 as opposed to the changed `0x39f36935e07f283cf1fd58984c3ed2e63e8b5ccd ` which only has a balance of 1.5